dataflow/src/main/java/org/checkerframework/dataflow/cfg/playground/ConstantPropagationPlayground.java

```diff
@@ -19,9 +20,9 @@ public static void main(String[] args) {
 
         // run the analysis and create a PDF file
         ConstantPropagationTransfer transfer = new ConstantPropagationTransfer();
-        // TODO: correct processing environment
-        Analysis<Constant, ConstantPropagationStore, ConstantPropagationTransfer> analysis =
-                new Analysis<>(transfer, null);
-        JavaSource2CFGDOT.generateDOTofCFG(inputFile, outputDir, method, clazz, true, analysis);
+        ForwardAnalysis<Constant, ConstantPropagationStore, ConstantPropagationTransfer>
+                forwardAnalysis = new ForwardAnalysisImpl<>(transfer);
+        CFGVisualizeLauncher.generateDOTofCFG(
```

Comment:
@wmdietl This PR will use `ForwardAnalysisImpl`, `ForwardAnalysis` which is created in #99. Should I separate it and ignore the related changes?

Reply:
You could then make this a PR into #99.